docs/asciidoc/static/configuration.asciidoc

```diff
@@ -258,14 +271,14 @@ output {
 [float]
 === Conditionals
 
-Sometimes you only want a filter or output to process an event under
-certain conditions. For that, you'll want to use a conditional!
+Sometimes you only want a plugin to process an event under
```

Comment:
In this case, I think 'filter or output' is more accurate. Conditionals are only valid for these plugin types and are not used in inputs or codecs.

Thoughts?